app/views/spree/admin/orders/bulk_management.html.haml

```diff
@@ -170,7 +170,7 @@
           %td.bulk
             %input{ :type => "checkbox", :name => 'bulk', 'ng-model' => 'line_item.checked', 'ignore-dirty' => true }
           %td.order_no{ 'ng-show' => 'columns.order_no.visible' } {{ line_item.order.number }}
-          %td.full_name{ 'ng-show' => 'columns.full_name.visible' } {{ line_item.order.full_name }}
+          %td.full_name{ 'ng-show' => 'columns.full_name.visible' } {{ line_item.order.last_name + ', ' + line_item.order.first_name }}
```

Comment:
I think we can continue to use `line_item.order.full_name` here.
`full_name` is slightly different.
```ruby
    def full_name
      "#{firstname} #{lastname}".strip
    end
```

Reply:
Actually, this was discussed in the issue and the requirement is to display "last_name, first_name" because this better represents what is being sorted. (Don't worry, I had to read the entire comment history to find this out).

However, if the first or last name is blank, this could look funny. I have another suggestion..